lesson_38/my-project/src/components/Product.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import './Product.css';
+
+export default function Product(props) {
```

Comment:
change `props` to `{ addProductToCart }` as we had on lessons
it's easier to understand what the input is for this component